client/components/faq/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+export default React.createClass( {
```

Comment:
This code definitely works as-is. But because this is a pure function (and if you handle the `heading` as a prop), you could also use the stateless function syntax here, which is kinda neat. 😄 
```js
import { PropTypes } from 'react';

const FAQ = ( { header, items } ) => {
	return (
		<div className="faq">
			<h1 className="faq__heading">{ header }</h1>
			<ul className="faq__list">
				{
					items.map( ( { question, answer }, ind ) =>
						<li className="faq__item" key={ `faq-${ ind }` }>
							<h4 className="faq__question">{ question }</h4>
							<p className="faq__answer">{ answer }</p>
						</li>
					)
				}
			</ul>
		</div>
	);
}

FAQ.propTypes = {
	items: PropTypes.arrayOf(
		PropTypes.shape( {
			question: PropTypes.string.isRequired,
			answer: PropTypes.node.isRequired
		} )
	).isRequired
}

export default FAQ;
```